plugins/modules/alt_disk.py

```diff
@@ -189,13 +189,11 @@ def get_free_pvs(module):
         if match_key and match_key.group(3) == 'None':
             hdisk = match_key.group(1)
             # Check if the disk has an _LVM signature using lquerypv
-            cmd = ['lquerypv', '-V', hdisk]
+            cmd = ['getlvodm', '-j', hdisk]
             ret, stdout, stderr = module.run_command(cmd)
```

Comment:
my concern here is:
If the goal is to figure out if the PV belongs to a volume group or still belongs to one, I do not recommend getlvodm.

why?
as the command implies, it gets the info from ODM and not the disk.

what could happen?
lets, say I created a VG
mkvg -f -y testvg hdisk1

now this, will create an entry in ODM and getlvodm works fine. however, let's say
i export the VG.

varyoffvg testvg
exportvg testvg

this will delete the entry in ODM but LVM metadata of the VG is still on the disk. getlvodm -j will
tell you that the PV does not belong to any VG but lquerypv -V will read from disk and tell you
that it belongs to a VG since there is still LVM metadata on disk

Reply:
then i think the lquerypv -V is the way to go. getlvodm only check ODM and not the disk. exported VG still has metadata on disk.
I was looking at the issue discussion back in the original repo, he was mentioning chpv -C which erases LVM ownership on the disk. 
so I am guessing that although lspv showed there was no VG on that disk, there was still LVM metadata hdisk1 that is why is did not work.
what could he have done to prep the disk was either like he said chpv -C, or when he still had a VG in on the disk and ODM, he probably should have ran reducevg instead of exportvg